File: API/Dtos/PassengerWriteDto.cs
using System;
using API.Infrastructure.Interfaces;

namespace API.Dtos {

    public class PassengerWriteDto : IBaseEntity {

        // PK
        public int Id { get; set; }
        // FKs
        public Guid? ReservationId { get; set; }
        public int GenderId { get; set; }
        public int NationalityId { get; set; }
        // Fields
        public string Lastname { get; set; }
        public string Firstname { get; set; }
        public string Birthdate { get; set; }
        public string SpecialCare { get; set; }
        public string Remarks { get; set; }

    }

}

File: FrontEnd/src/app/features/check-in/classes/dtos/passenger-write-dto.ts
import { Guid } from 'guid-typescript'

export interface PassengerWriteDto {

    reservationId: Guid
    genderId: number
    nationalityId: number
    occupantId: number
    lastname: string
    firstname: string
    birthdate: string
    remarks: string
    specialCare: string

}
